libs/domain/picking/src/utilities.ts

```diff
@@ -0,0 +1,9 @@
+export const formatTime = (time: Date): string => {
+  return time
+    .toLocaleString('en-US', {
```

Comment:
It is important to not hardcode the locale. We have a local service that can be used. Have a look a the `DefaultPricingService`. Also, the [Intl api](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat) might be useful to look at.

Reply:
The idea of using the current locale would be that we can use this app cross locales without coding changes

Reply:
Good idea! I think creating `DateTimeService` can also be reused in EB. It looks like a new feature and I think it can be separated into new subtask to avoid blocking Alvina and Pavel with the current PR